lib/biker.rb

```diff
@@ -1,3 +1,26 @@
 class Biker
+  attr_reader :name, :max_distance, :rides, :acceptable_terrain
 
-end
\ No newline at end of file
+  def initialize(name, max_distance)
+    @name = name
+    @max_distance = max_distance
+    @rides = Hash.new {|ride, time| hash[key] = []}
```

Comment:
This is an option, but just make sure you know what your block variables are doing here. You're creating them but not referencing the block variables. When in doubt, stick to the basics